documentation/guides/owner-journey.md

```diff
@@ -36,19 +42,25 @@ gwa login
 
 ## 3. Apply Configuration
 
-Templates are available for generating gateway configuration for popular integration patterns. Below has one that uses the [Client Credentials Oauth2 Grant](./tutorial-idp-client-cred-flow.md) and uses a [Shared Identity Provider](./tutorial-idp-client-cred-flow.md#shared-idp).
+Templates are available for generating gateway configuration for popular integration patterns. In this tutorial you will use a template to protect an API with an [Oauth 2.0 Client Credentials flow](./tutorial-idp-client-cred-flow.md) using a [Shared Identity Provider](./tutorial-idp-client-cred-flow.md#shared-idp).
+
+First, create a new Namespace. 
+
+```
+gwa namespace create -g
+```
 
-First, create a new namespace.
+`-g` generates a random, unique namespace which is displayed in response. Set the active namespace:
 
 ```
-gwa namespace create
+gwa config set --namespace <NEW-NAMESPACE>
```

Comment:
setting the namespace is not necessary as the create will set it automatically

Reply:
agreed, not needed if following these steps